ovs/extensions/generic/system.py

```diff
@@ -163,7 +163,7 @@ def get_free_ports(selected_range, exclude=None, nr=1, client=None):
         if client is None:
             output = check_output(cmd, shell=True)
         else:
-            output = client.run(cmd)
+            output = client.run(cmd.split(' '))
```

Comment:
Its better to use ```cmd.split()```, because ```split()``` will strip all whitespaces and ```split(' ')``` will keep whitespaces
Eg:
```
In [3]: cmd = 'cat  /proc/sys/net/ipv4/ip_local_port_range'   <-- 2 spaces

In [4]: cmd.split(' ')
Out[4]: ['cat', '', '/proc/sys/net/ipv4/ip_local_port_range']
```